pkgs/top-level/python-packages.nix

```diff
@@ -232,6 +232,8 @@ in {
 
   breathe = callPackage ../development/python-modules/breathe { };
 
+  brother_ql = callPackage ../development/python-modules/brother_ql { };
```

Comment:
Is `brother_ql` really a Python module, seems more like a CLI tool only. In that case, it doesn't belong in `pythonPackages` and should use `buildPythonApplication`.

Reply:
It is a library and application. There is a brother_ql_web program that uses brother_ql as a dependency. It has however also some scripts to create labels.

Reply:
Okay, then it belongs into `pythonPackages`.